### PR TITLE
Week 1

### PR DESCRIPTION
**Pull request by ryanmt95**
I've finished designing the main page of the website. And tinkered with a travel page. Currently facing some issues where the main section of the travel page appears underneath the background image, and z-index doesn't seem to fix the issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,5 @@ To get started:
   - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
 
 Then follow the on-screen instructions to set up your repo.
+
+Testing Modifications
```
Comment (lingxz) [inline README.md:14]: remove this?

File: portfolio/README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@ By default it contains a barebones web app. To run a local server, execute this
 command:
 
 mvn package appengine:run
+
+Mmodi
```
Comment (lingxz) [inline portfolio/README.md:8]: This also?

File: portfolio/src/main/webapp/assets/javascript/script.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+//   const greetings =
+//       ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+//   // Pick a random greeting.
+//   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+//   // Add it to the page.
+//   const greetingContainer = document.getElementById('greeting-container');
+//   greetingContainer.innerText = greeting;
+// }
```
Comment (lingxz) [inline portfolio/src/main/webapp/assets/javascript/script.js:28]: You can remove this, if you ever need it again you can still find it in version control

File: setup2.sh [hunk 4]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+gcloud config set project rtan-sps-summer20
+cd portfolio
```
Comment (lingxz) [inline setup2.sh:3]: Do you still need this file?

Comment (ryanmt95) [reply setup2.sh:3]: Hmmm yeah i guess not

**Comment by ryanmt95** [comment]
Deleted unnecessary code and files~

**Pull request merged**